includes/commands/chungus.js

```diff
@@ -730,6 +731,25 @@ function checkHeldTime(msg, args) {
 	});
 }
 
+function currentStreak(userId, callback){
+	db.get(
+		//There has to be a better way to only get the most recent row where "BecameChungus=1" but I don't know the db well enough to craft it.
+		`SELECT u.UserId, c.DateCreated, c.BecameChungus
+			FROM ChungusPoints c
+			INNER JOIN User u ON u.UserId = c.UserId
+			WHERE u.ServerId = (SELECT ServerId FROM User WHERE UserId = ${userId}) AND c.BecameChungus = 1
+			ORDER BY DateCreated DESC
+			LIMIT 1`,
+		function(err, row) {
+			let timeSinceBecameChungus = 0
+			if (row.UserId == userId) { //They are the current chungus leader -> timeSinceBecameChunges gets set to value != 0
```

Comment:
You can check for UserId in the row straight in the SQL. (See my suggested query)

If the row returned is undefined, then you know they have never been chungus

Reply:
Thanks, I will definitely simplify this.
However, I wanted to return a zero in the callback if they are not currently chungus.
If I understand correctly, your query will give the row of when they last became chungus, while I intended to retrieve the row of who most recently became chungus and compare that to the called user in "stats."

Reply:
Ah yeah you're totally right. I spaced thats what you were wanting to accomplish here.

Well if thats the case, you do need that server bit of SQL in the code in order to make sure you only select users from the current server. You might still be able to simplify it, but Im not sure how. Its also not a big deal to optimize it, as this code is not called often, and its not a highly trafficked bot anyways.

If you just revert that SQL change it should be good.